examples/action_mailer.rb

```diff
@@ -1,11 +1,16 @@
 # This gem adds ActionMailer delivery method.
 # To configure it, add following to your ActionMailer configuration
-# (in Rails projects located in `config/$ENVIRONMENT.rb`)
+# (in Rails projects located in `config/environments/production.rb`)
 config.action_mailer.delivery_method = :mailtrap
+
+# Customize the settings:
 config.action_mailer.mailtrap_settings = {
-  api_key: ENV.fetch('MAILTRAP_API_KEY'),
-  # bulk: true, # Bulk sending (@see https://help.mailtrap.io/article/113-sending-streams)
-  # sandbox: true, inbox_id: 12, # Sandbox sending (@see https://help.mailtrap.io/article/109-getting-started-with-mailtrap-email-testing)
+  # Use custom API key (not necessary if you've set the MAILTRAP_API_KEY environment variable)
+  api_key: Rails.application.credentials.my_mailtrap_api_key!,
+  # Switch to bulk sending (@see https://help.mailtrap.io/article/113-sending-streams)
+  bulk: true,
+  # Switch to sandbox sending (@see https://help.mailtrap.io/article/109-getting-started-with-mailtrap-email-testing)
+  sandbox: true, inbox_id: 12,
```

Comment:
Would it make sense to leave these options commented out though? Just so that somebody doesn't accidentally leave some of them uncommented after copy paste

```suggestion
  # api_key: Rails.application.credentials.my_mailtrap_api_key!,
  # Switch to bulk sending (@see https://help.mailtrap.io/article/113-sending-streams)
  # bulk: true,
  # Switch to sandbox sending (@see https://help.mailtrap.io/article/109-getting-started-with-mailtrap-email-testing)
  # sandbox: true, inbox_id: 12,
```

Reply:
Well... OK. I've considered this too. My argument against was that this entire code is unnecessary, you only need `delivery_method = :mailtrap` unless you are doing something special. :)

Reply:
You can't be too safe against blind copy-paste i suppose. :)